addActivity.js

```diff
@@ -0,0 +1,15 @@
+export const activities = [];
+
+const productivityScores = {
+    'Work': 10,
+    'Exercise': 8,
+    'Leisure': 5,
+    'Learning': 9,
+    'Other': 4
+};
+
+export function addActivity(name, category, duration) {
```

Comment:
Hey @Merongk, as an extra exercise, could you add a sanity check here? Let's say a user is trying to add an activity with a category that does not exist.

<details>
  <summary>Hint</summary>
  You can use `Object.keys` 
</details>